docker/experimental/tools.Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     wget \
     curl \
     perl-base=5.32.1-4+deb11u1 \
+    libtinfo6=6.2+20201114-2+deb11u1 \
```

Comment:
who is going to update these? (when do they become stale?)

cc @gedigi

Reply:
I tried removing the pin in perl-base before this, and the install failed with another unresolved dependency on libncursesw. My guess is that either python3-botocore/bullseye or awscli/bullseye depends on older packages and is not getting rebuilt frequently to keep up with Debian.
This is definitely a bad thing and we should try to avoid it but on the short term this is the best we can do.